mtags/src/main/scala/scala/meta/internal/mtags/MD5Java.java

```diff
@@ -0,0 +1,39 @@
+
+package scala.meta.internal.mtags;
+
+import java.nio.CharBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/** Java semanticdb plugin returns a different md5 hashsum than the Scala one
+  * Copied over from:
+  * https://github.com/sourcegraph/lsif-java
+  */
+public final class MD5Java {
```

Comment:
This is a fun one, so it seems the MD5 produced by the Java implementation and Scala one is different! Why? I cannot figure that out :scream:

Reply:
Could you check which implementation is correct using `md5sum` in the terminal?

Reply:
Good idea, it seems it's the Scala one that returns the same MD5 string.

Reply:
Reported upstream https://github.com/sourcegraph/lsif-java/issues/391